modyn/selector/internal/selector_strategies/random_downsampling_strategy.py

```diff
@@ -0,0 +1,7 @@
+from modyn.selector.internal.selector_strategies.abstract_downsample_strategy import AbstractDownsampleStrategy
+
+
+class RandomDownsamplingStrategy(AbstractDownsampleStrategy):
```

Comment:
Can you please add a few tests that this class has its expected behavior (probably just copy/paste some tests from AbstractDownsampleStrategy)?